src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+
+* {
+  margin: 0;
```

Comment:
set only `box-sizing: border-box` in `*` selector

but margin should be reset for list of tags you really need